docs/platform/howto/pro-platform.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Pro Platform
+---
+
+import EarlyBadge from "@site/src/components/Badges/EarlyBadge";
+
+<EarlyBadge/>
+
+Pro Platform gives enterprise organizations added capabilities for implementing security resilience, reliability, and cost-efficiency strategies for their data infrastructure.
+
+Pro Features are available across disaster recovery, security and compliance,
+advanced networking, and user lifecycle management.
+
+To enable Pro Platform for your organization and use Pro features, contact the sales team
+at sales@aiven.io. You must be a super admin to request this.
```

Comment:
We should collectively agree on the "contact the sales team" syntax: I've shortened things in many places to `[contact the Sales team](mailto:sales@aiven.io)` to reduce word count and avoid having email addresses in the content, wdyt? 

Also perhaps you can use your new admonition here (or maybe it's not applicable, but if we can, why not)?

Reply:
Yep, totally fine with that. Perhaps another use case for a snippet so we all use the same thing every time?

Regarding the admonition: yes, that was plan once that PR gets merged. 👍